seqr/utils/elasticsearch/es_search.py

```diff
@@ -762,7 +764,7 @@ def _parse_compound_het_response(self, response):
                 compound_het_pairs_by_gene[gene_id] = gene_compound_het_pairs
 
         total_compound_het_results = sum(len(compound_het_pairs) for compound_het_pairs in compound_het_pairs_by_gene.values())
-        logger.info('Total compound het hits: {}'.format(total_compound_het_results))
+        logger.info('Total compound het hits: {}'.format(total_compound_het_results), extra={'user': self._user})
```

Comment:
It seems not a necessary log.

Reply:
I like having the detailed breakdown for search results in the logs, we get lots of search related issues so its helpful to see exactly what is happening